src/index.ts

```diff
@@ -22,10 +23,14 @@ export default (app: Probot) => {
   // on receive star event
   app.on("star.created", async (context: Context<"star.created">) => {
     var payload = context.payload.repository;
-    var msg = `Repo: ${payload.name} received a new star! Total stars: ${payload.stargazers_count}`;
-    app.log.info(msg);
+    const actualStars = (await kv.get(`${payload.name}.stars`)) as number;
```

Comment:
You want to wrap the string value with `Number.parseInt` to parse it as number rather than casting the type.

```suggestion
    const actualStars = Number.parseInt(await kv.get(`${payload.name}.stars`));
```